index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Setup Mobile Project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <article>
+        <header>   
+            <time class="time"><b>9:41</b></time>
+        <span>
+            <img class="menu-icon" src="/images/Icon - Menu.svg" alt="hamburger list">
+            <img class="pwr" src="/images/Battery.svg" alt="battery">
+            <img class="wifi" src="/images/Wifi.svg" alt="wifi">
+            <img class="cell-connection" src="/images/Cellular Connection.svg" alt="cell connection">
+        </span>
+            <p class="logo">My Logo</p>  
+            <h2>I'm Mariane Glad to see you!</h2>
+            <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        </header>
+    </article>  
```

Comment:
- For smenatic reasons, You do not need to use an `article` tag here, the `header` tag does it 👌. Please fix this 👍

Reply:
Done, thank you for the feedback.